src/script/modules/send-message-with-content/index.ts

```diff
@@ -0,0 +1,47 @@
+import settings from '../../lib/settings';
+import { registerMiddleware, updateSnapchatStore } from '../../utils/middleware';
+import { getSnapchatStore } from '../../utils/snapchat';
+
+let oldGetConversationManager: any = null;
+
+function handleStoreEffect(storeState: any) {
+  if (!storeState.messaging?.client) {
+    return storeState;
+  }
+
+  if (!oldGetConversationManager) {
+    oldGetConversationManager = storeState.messaging.client.getConversationManager;
```

Comment:
we set `oldGetConversationManager` but never set it back when the setting is unset?

Reply:
ideally only wanna patch a method when a setting requires it